src/app/views/query-response/query-response.scss

```diff
@@ -0,0 +1,13 @@
+@import "../../../styles/variables";
+
+.query-response {
+  display: flex;
+  margin-top: $gutter;
+  width: 100%;
+  height: 375px;
+
+  .ms-Pivot.root-84 {
```

Comment:
Why were these styles removed?

Reply:
Adding them back with a few modifications to make them easy on the eyes